app/plugin_api_tests/manual.test_http_hijack_plugin/main.go

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package main
+
+import (
+	"net/http"
+
+	"github.com/mattermost/mattermost-server/v5/plugin"
+)
+
+type Plugin struct {
+	plugin.MattermostPlugin
+}
+
+func (p *Plugin) ServeHTTP(c *plugin.Context, w http.ResponseWriter, r *http.Request) {
+	hj, ok := w.(http.Hijacker)
+	if !ok {
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+
+	conn, brw, err := hj.Hijack()
```

Comment:
nit: `brw` vs. the example of `bufrw` in the https://golang.org/pkg/net/http/#Hijacker example? No objection to the abbreviating (I was expecting it given the context), but wondering if it would boost readability to align with the docs.

Reply:
Yes, since it's quite an internal detail I didn't spend too time much on naming. 
We can surely align with the docs though :+1: